Classes/Domain/Repository/TokenRepository.php

```diff
@@ -42,7 +42,7 @@ public function deleteExpiredTokens(int $expireTime): void
         $constraints[] = $query->lessThan('tstamp', (int) (time() - $expireTime));
 
         if (count($constraints)) {
-            $query->matching($query->logicalAnd($constraints));
+            $query->matching($query->logicalAnd(...array_values($constraints)));
```

Comment:
@thomaslow What is the reason behind this change? `$constraints` is already an array so this change looks not necessary.

Reply:
@beatrycze-volk The `...` (unpacking of array) change was necessary due to the API change described in [Breaking #96044](https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.0/Breaking-96044-HardenMethodSignatureOfLogicalAndAndLogicalOr.html). I am not sure whether `array_values()` is really necessary.

Reply:
Thanks for answer. I would suspect that only `...` are needed, as array is already declared.

Reply:
As per documentation `array_values` function returns `array`:
array_values([array](https://www.php.net/manual/en/language.types.array.php) $array): [array](https://www.php.net/manual/en/language.types.array.php)

Then I will remove call of `array_values` and make PR for it.